operator/validator/router.go

```diff
@@ -1,34 +1,36 @@
 package validator
 
 import (
-	spectypes "github.com/bloxapp/ssv-spec/types"
 	"go.uber.org/zap"
 
 	"github.com/bloxapp/ssv/network/commons"
+	"github.com/bloxapp/ssv/protocol/v2/ssv/queue"
 )
 
 const bufSize = 1024
 
-func newMessageRouter() *messageRouter {
+func newMessageRouter(logger *zap.Logger) *messageRouter {
 	return &messageRouter{
-		ch:    make(chan spectypes.SSVMessage, bufSize),
-		msgID: commons.MsgID(),
+		logger: logger,
+		ch:     make(chan *queue.DecodedSSVMessage, bufSize),
+		msgID:  commons.MsgID(),
 	}
 }
 
 type messageRouter struct {
-	ch    chan spectypes.SSVMessage
-	msgID commons.MsgIDFunc
+	logger *zap.Logger
+	ch     chan *queue.DecodedSSVMessage
+	msgID  commons.MsgIDFunc
 }
 
-func (r *messageRouter) Route(logger *zap.Logger, message spectypes.SSVMessage) {
+func (r *messageRouter) Route(logger *zap.Logger, message *queue.DecodedSSVMessage) {
```

Comment:
logger is now not used, should we take it out? I know its a pain in the a** to do so though. also we might want to pass in context instead, this way we can cancel the send and we can attach some data to it in the future.

Reply:
Done